docs/_templates/search.html

```diff
@@ -15,173 +15,13 @@
     </div>
 
     <nav id="pagination" class="pagination" aria-label="pagination" hidden="true" >
-        <button class="pagination-previous" id="pagination-previous">
-            ← Previous
-        </button>
-        <ul class="pagination-list"></ul>
-        <button class="pagination-next" id="pagination-next">
-            Next →
-        </button>
+        <div class="pagination-previous" id="pagination-previous"></div>
+        <div class="pagination-item"></div>
+        <div class="pagination-next" id="pagination-next"></div>
```

Comment:
if `pagination-previous` and `pagination-next` are still acting as buttons (from what I can see they are), they should still be buttons in the DOM.

```suggestion
        <button class="pagination-next" id="pagination-next" type="button" data-pagination-next></button>
```
also, while we are changing this, it might be nicer to pick up the JS section via a data attribute `data-pagination-next` as per our current UI guidelines.

https://docs.wagtail.org/en/latest/contributing/ui_guidelines.html#html-guidelines